src/internal/lib/connectors/services/water/NotificationsApiClient.js

```diff
@@ -11,7 +11,7 @@ class NotificationsApiClient extends SharedNotificationsApiClient {
    * @param {String} sender - email address of sender.  If not supplied, reverts to preview mode
    * @return {Promise} resolves with an array of contacts, each with licence numbers and rendered templates attached
    */
-  sendNotification (taskConfigId, licenceNumbers, params = {}, sender = null) {
+  sendNotification (taskConfigId, licenceNumbers, uniqueJobId, params = {}, sender = null) {
```

Comment:
Someone went to the effort of adding JSDoc comments for the existing params. So, I don't think it would hurt to update them with this new one.